tests/fsaptd-terms/input.dat

```diff
@@ -0,0 +1,87 @@
+#! F-SAPT0/jun-cc-pvdz procedure for methane dimer 
+
+memory 1 GB
+
+molecule mol {
+0 1
+C 0.00000000 0.00000000 0.00000000
+H 1.09000000 0.00000000 0.00000000
+H -0.36333333 0.83908239 0.59332085
+H -0.36333333 0.09428973 -1.02332709
+H -0.36333333 -0.93337212 0.43000624
+--
+0 1
+C 6.44536662 -0.26509169 -0.00000000
+H 7.53536662 -0.26509169 -0.00000000
+H 6.08203329 0.57399070 0.59332085
+H 6.08203329 -0.17080196 -1.02332709
+H 6.08203329 -1.19846381 0.43000624
+symmetry c1
+no_reorient
+no_com
+}
+
+set {
+basis         jun-cc-pvdz
+scf_type df
+guess sad
+freeze_core true
+}
+
+energy('fisapt0-d3m')
+
+keys = ['Enuc', 'Eelst', 'Eexch', 'Eind', 'Edisp', 'Eedisp', 'Etot']  #TEST
+
+Eref = {  #TEST
+    'Enuc'  : 35.07529824960602,        #TEST
+    'Eelst' : -3.8035153870907834e-06,  #TEST
+    'Eexch' : 1.7912112685446533e-07,   #TEST
+    'Eind'  : -3.833795151474493e-08,   #TEST
+    'Edisp' :  0.000000000000,          #TEST
+    'Eedisp': -0.00005312000,           #TEST
+    'Etot'  : -0.00005678273
+    }  #TEST
+
+
+Epsi = {  #TEST
+    'Enuc'  : mol.nuclear_repulsion_energy(),           #TEST
+    'Eelst' : variable("SAPT ELST ENERGY"),    #TEST
+    'Eexch' : variable("SAPT EXCH ENERGY"),    #TEST
+    'Eind'  : variable("SAPT IND ENERGY"),     #TEST
+    'Edisp' : variable("SAPT DISP ENERGY"),    #TEST
+    'Eedisp' : variable("FISAPT0-D DISP ENERGY"),    #TEST
```

Comment:
Is this the you/CDS/Dom consensus, Jeff, that `SAPT DISP ENERGY` be defined but 0 and `[FI]SAPT0-D DISP ENERGY` be nonzero for these calcs? I don't have a better proposal, so just checking that this isn't going to surprise anyone.

Reply:
Not so much a consensus, but what I thought made sense. Happy to change it if people feel strongly about alternatives.  Also, thanks for pointing out the CMakeLists.txt labels, I'll be pushing some cleanup to tests in a moment

Reply:
I think it makes sense for `SAPT0 DISP ENERGY` to be 0, and `SAPT0-D DISP ENERGY` to be nonzero here, but the corresponding `SAPT0 TOTAL ENERGY` and `SAPT0-D TOTAL ENERGY` would be necessarily different. I think that's the consistent way (and how it is here), but users may want `SAPT0 TOTAL ENERGY` to include the empirical dispersion.